assembly/index.ts

```diff
@@ -1,3 +1,33 @@
+const welcomePage = `<!DOCTYPE html>
```

Comment:
We should keep this consistent (if possible, cc @torch2424) with the JS version https://github.com/fastly/compute-starter-kit-javascript-default/blob/main/src/index.js where we import the content from disk.